dist/40-10007-schema.json

```diff
@@ -305,305 +305,299 @@
     }
   },
   "properties": {
-    "applications": {
-      "type": "array",
-      "items": {
```

Comment:
Should we use the schema
```
   "properties": {
      "application": {
        "type": "object",
        "required": { "applicant", "claimant", ....
```

Reply:
The whole thing will be the application, so I don't think there is a point to wrapping everything, unless it's a lot easier to do so for the backend. I can add the required line in either case.

Reply:
I think we should probably do the wrap, if it would be too involved on the FE. Its a resource, and should we resurrect the concept of multiple applications, the structure of { application: ... one or more applications ... } would already be in place - and it does simplify the BE a tad.

Reply:
Done.